src/pages/pt-br/guides/deploy.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Faça deploy do seu site Astro
+description: Como fazer deploy do seu site Astro para a web.
+layout: ~/layouts/MainLayout.astro
+setup: import DeployGuidesNav from '~/components/DeployGuidesNav.astro';
+i18nReady: true
+---
+**Pronto para fazer a build e deploy do seu site Astro?** Siga um dos nossos guias para diferentes serviços de deploy ou role para abaixo para um guia geral sobre como fazer deploy de um site Astro.
+
+<DeployGuidesNav />
+
+## Opções de Deploy Rápido
+
+Você pode fazer build e deploy de um site Astro para diversos hosts rapidamente utilizando seja pela UI do painel dessa plataforma ou por uma interface de linha de comando.
+
+### UI do Website
+
+Uma forma rápida de fazer deploy do seu website é conectando o repositório Git online (e.x. GitHub, GitLab, Bitbucket) de seu projeto Astro a uma provedor de hospedagem e se aproveitando de continuous deployment utilizando o Git.
+
+Estas plataformas de host automaticamente detectam pushes no repositório fonte do seu projeto Astro, fazem a build do seu site e então fazem o deploy dele para a web em uma URL customizada ou em seu domínio pessoal. Geralmente, para configurar o deployment nessas plataformas você vai acabar seguindo passos como os seguintes: 
+
+1. Adicione seu repositório para um provedor Git online (e.x. GitHub, GitLab, Bitbucket)
+
+1. Escolha uma host que suporta **continuous deployment** (e.x. [Netlify]/pt-br/guides/deploy/netlify/) ou [Vercel](/pt-br/guides/deploy/vercel/)) e importe seu repositório Git como um novo site/projeto.
```

Comment:
all ordered lists are repeating the number one, does this get transformed on the final page?

Reply:
Yes! They're rendered properly. Also, thanks for highlighting this line just found a broken link, going to fix this real quick 😅

Reply:
:sweat_smile: no problems!